tests/test_logger.py

```diff
@@ -147,24 +147,19 @@ def test_logging_loop_add_to_queue(self, logger):
     # This decorator is used to run the same test with different data
     # read more about it here: https://docs.pytest.org/en/stable/parametrize.html
     @pytest.mark.parametrize(
-        "data_packet",
+        ("state", "extension", "imu_data_packets", "processed_data_packets"),
         [
-            LoggedDataPacket(
-                **{k: "1" for k in LoggedDataPacket.__struct_fields__},
-            ),
+            ("S", 0.0, [RawDataPacket(0.0)], [ProcessedDataPacket(0.0, 0.0)]),
```

Comment:
please initalize the packet with all the fields, like was done previously.

Reply:
I want to go to bed now so I will fix this later lol

Reply:
im fixing